src/View/Form/EntityContext.php

```diff
@@ -147,7 +148,7 @@ protected function _prepare()
             $table = TableRegistry::get($table);
         }
 
-        if (!is_object($table)) {
+        if (!($table instanceof RepositoryInterface)) {
```

Comment:
Could this be a BC break for some apps?

Reply:
The object comes from `TableRegistry::get()` and later there's a call `$table->getAlias()`, so extremely unlikely it's not going to be an `RepositoryInterface` instance.